apps/etterlatte-saksbehandling-ui/client/package.json

```diff
@@ -22,10 +22,10 @@
     "react": "^18.2.0",
     "react-datepicker": "^4.8.0",
     "react-dom": "^18.2.0",
-    "react-hook-form": "^7.39.1",
+    "react-hook-form": "^7.41.5",
     "react-redux": "^8.0.5",
     "react-router": "^6.4.3",
```

Comment:
Usikker på om det hører til denne PR-en, men trenger vi egentlig `react-router` når vi har `react-router-dom`? Jeg trodde sistnevnte inkluderte alt, pluss litt til. Og hvis den trengs, bør versjonene være i sync?

Reply:
Dei bør vera i synk, vi fekk feil her for nokre dagar sia da vi merga inn berre den eine ⚠️

Reply:
kan være at vi klarer oss kun med `react-router-dom` som @EspenSchei sier. react-router er core paketet men react-router-dom tar in den som dependency og eksporterer det vi trenger. ja blir sikkert noe feil hvis vi har begge de paketene men i ulike versjoner.

Reply:
Kan undersøke! Jeg merger inn PRen nå først uansett for å bli kvitt Snyk-PRene, men forsøker å fjerne dependencien i en ny pr 👍